http/interceptor.go

```diff
@@ -103,7 +104,7 @@ func wrap(w http.ResponseWriter, r *http.Request, tx types.Transaction) (
 			// we release the buffer
 			reader, err := tx.ResponseBodyReader()
 			if err != nil {
-				i.w.WriteHeader(http.StatusInternalServerError)
+				w.WriteHeader(http.StatusInternalServerError)
```

Comment:
Curious about the reason for this change

Reply:
Inside the same function, we are using both `i.w.WriteHeader` and `w.WriteHeader`. I thought it was good to make all of them the same (with the latest commit, all `i.w.WriteHeader`)